kubectl minio proxy : panic: runtime error: index out of range [0] with length 0

failed to start minio proxy : `panic: runtime error: index out of range [0] with length 0`

```
root@k8s-master-1:~# kubectl version --output=yaml
clientVersion:
  buildDate: "2022-05-03T13:46:05Z"
  compiler: gc
  gitCommit: 4ce5a8954017644c5420bae81d72b09b735c21f0
  gitTreeState: clean
  gitVersion: v1.24.0
  goVersion: go1.18.1
  major: "1"
  minor: "24"
  platform: linux/amd64
kustomizeVersion: v4.5.4
serverVersion:
  buildDate: "2022-05-03T13:38:19Z"
  compiler: gc
  gitCommit: 4ce5a8954017644c5420bae81d72b09b735c21f0
  gitTreeState: clean
  gitVersion: v1.24.0
  goVersion: go1.18.1
  major: "1"
  minor: "24"
  platform: linux/amd64

root@k8s-master-1:~# kubectl minio version
v4.4.18
root@k8s-master-1:~# kubectl minio proxy -n minio-operator
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/minio/kubectl-minio/cmd.(*operatorProxyCmd).run(0xc000182000)
        github.com/minio/kubectl-minio/cmd/proxy.go:91 +0x597
github.com/minio/kubectl-minio/cmd.newProxyCmd.func1(0xc0000bf8c0, {0xc000198500, 0x2, 0x2})
        github.com/minio/kubectl-minio/cmd/proxy.go:59 +0x25
github.com/spf13/cobra.(*Command).execute(0xc0000bf8c0, {0xc0001984c0, 0x2, 0x2})
        github.com/spf13/cobra@v1.1.1/command.go:850 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0x265ab20)
        github.com/spf13/cobra@v1.1.1/command.go:958 +0x3ad
github.com/spf13/cobra.(*Command).Execute(...)
        github.com/spf13/cobra@v1.1.1/command.go:895
main.main()
        github.com/minio/kubectl-minio/main.go:30 +0x97
root@k8s-master-1:~# kubectl minio proxy
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/minio/kubectl-minio/cmd.(*operatorProxyCmd).run(0xc0000d2000)
        github.com/minio/kubectl-minio/cmd/proxy.go:91 +0x597
github.com/minio/kubectl-minio/cmd.newProxyCmd.func1(0xc00021cb00, {0x26a0c58, 0x0, 0x0})
        github.com/minio/kubectl-minio/cmd/proxy.go:59 +0x25
github.com/spf13/cobra.(*Command).execute(0xc00021cb00, {0x26a0c58, 0x0, 0x0})
        github.com/spf13/cobra@v1.1.1/command.go:850 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0x265ab20)
        github.com/spf13/cobra@v1.1.1/command.go:958 +0x3ad
github.com/spf13/cobra.(*Command).Execute(...)
        github.com/spf13/cobra@v1.1.1/command.go:895
main.main()
        github.com/minio/kubectl-minio/main.go:30 +0x97
root@k8s-master-1:~#
```

--- a/helm/operator/templates/console-ui.yaml
+++ b/helm/operator/templates/console-ui.yaml
@@ -4,6 +4,15 @@ metadata:
   name: console-sa
   namespace: {{ .Release.Namespace }}
 ---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: console-sa-secret
+  namespace: minio-operator
+  annotations:
+    kubernetes.io/service-account.name: console-sa
+type: kubernetes.io/service-account-token
+---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:

--- a/kubectl-minio/cmd/proxy.go
+++ b/kubectl-minio/cmd/proxy.go
@@ -88,7 +88,11 @@ func (o *operatorProxyCmd) run() error {
 	if err != nil {
 		return err
 	}
-	secretName := sa.Secrets[0].Name
+
+	secretName := "console-sa-secret"
+	if len(sa.Secrets) > 0 {
+		secretName = sa.Secrets[0].Name
+	}
 
 	secret, err := client.CoreV1().Secrets(o.operatorOpts.Namespace).Get(ctx, secretName, metav1.GetOptions{})
 	if err != nil {

--- a/kubectl-minio/cmd/resources/common.go
+++ b/kubectl-minio/cmd/resources/common.go
@@ -16,7 +16,6 @@
 package resources
 
 import (
-	"fmt"
 	"io/fs"
 	"io/ioutil"
 	"log"
@@ -192,11 +191,11 @@ func copyDirtoMemFS(src string, dst string, memFS filesys.FileSystem) error {
 
 		if fd.IsDir() {
 			if err = copyDirtoMemFS(srcfp, dstfp, memFS); err != nil {
-				fmt.Println(err)
+				return err
 			}
 		} else {
 			if err = copyFileToMemFS(srcfp, dstfp, memFS); err != nil {
-				fmt.Println(err)
+				return err
 			}
 		}
 	}

--- a/resources/base/console-ui.yaml
+++ b/resources/base/console-ui.yaml
@@ -4,6 +4,15 @@ metadata:
   name: console-sa
   namespace: default
 ---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: console-sa-secret
+  namespace: minio-operator
+  annotations:
+    kubernetes.io/service-account.name: console-sa
+type: kubernetes.io/service-account-token
+---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:

